# Teaching Overview


## Teaching at TU Braunschweig
PuckerLab website: [Pucker lab website at TU Braunschweig](https://www.tu-braunschweig.de/en/ifp/pbb)

### MB09 - Introduction to Biochemistry and Bioinformatics of Plants
Material used in this course can be found [here](https://github.com/bpucker/teaching/tree/master/MB09_PlantBiochemistryBioinformatics).

### GE31/MM12 - Applied Plant Genomics
Material used in this course can be found [here](https://github.com/bpucker/teaching/tree/master/GE31_AppliedPlantGenomics). Scripts and instructions of the bioinformatics component are available [here](https://github.com/bpucker/AppliedPlantGenomics).

### GE32/MM12 - Data Literacy in Plant Sciences
Material used in this course can be found [here](https://github.com/bpucker/teaching/tree/master/GE32_DataLiteracyInPlantSciences). Scripts and instructions of the bioinformatics component are available [here](https://github.com/bpucker/DataLiteracyInPlantSciences).

### BB34 - Python for Life Scientists
Material used in this course can be found [here](https://github.com/bpucker/PythonTUBS). Additional data sets are available [here](https://cloud.tu-braunschweig.de/s/HXrzS4pFrLxdAMb).

### FRX - Data Literacy in Genome Research / Genomics
Material used in this course can be found [here](https://github.com/bpucker/teaching/tree/master/FRX_DataLiteracyInGenomeResearch).

### TUBS - Applied Plant Transcriptomics (in preparation)
Material used in this course can be found [here](https://github.com/bpucker/AppliedPlantTranscriptomics).


### iGEM @ TU Braunschweig ###
[PuckerLab](https://www.tu-braunschweig.de/en/ifp/pbb) is hosting the [TU Braunschweig iGEM team 2022](https://www.tu-braunschweig.de/en/igem/2022).

<a href="https://www.tu-braunschweig.de/en/igem/2022">
<div>
<img alt="iGEM" src="https://github.com/bpucker/figures/blob/main/igem_logo.png" width="20%" height="20%">
</div>
</a>  


## Teaching at Bielefeld University (CeBiTec)

### Applied Python Programming for Life Scientists
The intention of this course is to enable life scientists with different backgrounds to solve their own data analysis issues by writing Python scripts for the automatic processing. Evaluation results of this course are discussed [here](https://pub.uni-bielefeld.de/download/2920633/2920815). All materials are available for re-use (CC-BY 4.0).

[Applied Python Programming for Life Scientists - github repository](https://github.com/bpucker/APPLS)

<a href="https://github.com/bpucker/APPLS">
<img alt="APPLS" src="https://github.com/bpucker/figures/blob/main/Python_course.png" width="50%" height="50%">
</a>  




### Applied Genome Research
This course provides an overview of bioinformatic methods frequently used in genome research. Participants run all analysis on real data sets. Evaluation results of this course are discussed [here](https://doi.org/10.1515/jib-2019-0005). All materials are available for re-use (CC-BY 4.0).

[Applied Genome Research - github repository](https://github.com/bpucker/AppliedGenomeResearch)


<a href="https://github.com/bpucker/AppliedGenomeResearch">
<img alt="Applied Genome Research course content overview" src="https://github.com/bpucker/figures/blob/main/AppliedGenomeReserachCourseContentOverview.png" width="100%" height="100%">
</a>  



### Molecular Methods in Genome Research
This course provides an overview of molecular biology methods frequently used in genome research. Participants run all analysis on real data sets. Each participant is performing a different experiment. Evaluation results of this course are discussed [here](https://doi.org/10.1515/jib-2019-0005). All materials are available for re-use (CC-BY 4.0).

[Molecular Methods in Genome Research - github repository](https://github.com/bpucker/MolecularMethodsInGenomeResearch)


<a href="https://github.com/bpucker/MolecularMethodsInGenomeResearch">
<img alt="Molecular Methods in Genome Research course content overview" src="https://github.com/bpucker/figures/blob/main/MolecularMethodsInGenomeReserachCourseOverview.png" width="50%" height="50%">
</a>  



### Synthetic Biology (iGEM): Bielefeld-CeBiTec 

iGEM is the biggest synthetic biology competition for students. Teams from Bielefeld are participating since 2010. Detailed information about iGEM at the Center for Biotechnology (CeBiTec) at Bielefeld University is available here:

[iGEM Bielefeld-CeBiTec](https://igem-bielefeld.de/en/)

<a href="https://igem-bielefeld.de/en/">
<div>
<img alt="iGEM" src="https://github.com/bpucker/figures/blob/main/igem_logo.png" width="20%" height="20%">
<img alt="CeBiTec" src="http://2011.igem.org/wiki/images/e/ec/Bielefeld-Germany2011-CeBiTec.jpg" width="30%" height="30%">
</div>
</a>  


### Bielefelder certificate of higher education
In 2017, I received the Bielefelder Certificate in Higher Education. Details are available [here](https://www.uni-bielefeld.de/einrichtungen/zll/hdle/fortbildungen/zertifikat/) (in German). A description of my teaching project is available [here](http://docplayer.org/123503344-Angewandte-genomforschung-ngs-datenanalyse-fuer-biowissenschaftlerinnen.html) (in German).

<img alt="higher education certificate" src="https://github.com/bpucker/figures/blob/main/higher_education_certificate.png" width="50%" height="50%">




### Fellowship Digital innovations in higher education
A fellowship provided by the Stifterverband supported the development of a digital lab book for practical courses in the life sciences. 'Molecular Methods in Genome Research' was implemented as part of this fellowship. A detailed project description can be found [here](https://www.stifterverband.org/digital-lehrfellows/2017/pucker) (in German).

<a href="https://www.stifterverband.org/digital-lehrfellows/2017/pucker">
<img alt="Stifterverband Logo" src="https://upload.wikimedia.org/wikipedia/commons/d/d0/Logo_Stifterverband_%284C%29_.jpg" width="20%" height="20%">
<img alt="Exzellenz in der Lehre Logo" src="https://www.stifterverband.org/sites/default/files/styles/1280x720_c70/public/exzellenz_in_der_lehre_logo.jpg?itok=IVJXjA4R" width="30%" height="30%">
</a>


# References

## Teaching publications

Friedrich, A., & **Pucker, B.** (2018). Peer-review as a teaching method. Working Paper der AG Forschendes Lernen in der dghd, 2, 2018. Carl von Ossietzky Universität Oldenburg.[PDF](https://pub.uni-bielefeld.de/download/2920633/2920815)


**Pucker, B.** Schilbert, H.M., Schumacher, S.F. (2019). Integrating Molecular Biology and Bioinformatics Education. Journal of Integrative Bioinformatics. doi:[10.1515/jib-2019-0005](https://doi.org/10.1515/jib-2019-0005).


## Data set publications

**Pucker, B.**, Holtgräwe, D., Rosleff Sörensen, T., Stracke, R., Viehöver, P., and Weisshaar, B. (2016). A de novo Genome Sequence Assembly of the Arabidopsis thaliana Accession Niederzenz-1 Displays Presence/Absence Variation and Strong Synteny. PloS-ONE 11:e0164321. doi:[10.1371/journal.pone.0164321](https://doi.org/10.1371/journal.pone.0164321).


Haak, M., Vinke, S., Keller, W., Droste, J., Rückert, C., Kalinowski, J., & **Pucker, B.** (2018). High Quality de novo Transcriptome Assembly of Croton tiglium. Frontiers in Molecular Biosciences, 5. doi:[10.3389/fmolb.2018.00062](https://doi.org/10.3389/fmolb.2018.00062).

