# Course1: Plant Genome Sequence Assembly and Annotation (Mon-Wed, 08th - 10th Dec)
- VM access
- read statistics
- read correction (HERRO)
- assembly (Shasta)
- assembly evaluation
- BUSCO
- RNA-seq mapping (HISAT2/STAR)
- annotation with GeMoMa


# Course2: From Gene Models to Biological Insights - Functional Annotation (Wed-Fri, 10th -12th Dec)
- VM access
- BUSCO
- TE annotation
- InterProScan
- tRNA-scanSE
- KIPEs
- MYB_annotator
- tree building
- tree visualization
- gene duplication analysis


## References
- de Oliveira, J. A. V. S.; Choudhary, N.; Meckoni, S. N.; Nowak, M. S.; Hagedorn, M.; Pucker, B. (2025). Cookbook for Plant Genome Sequences. doi: [10.20944/preprints202508.1176.v2](https://www.doi.org/10.20944/preprints202508.1176.v2).
- Pucker B, Irisarri I, de Vries J and Xu B (2022). Plant genome sequence assembly in the era of long reads: Progress, challenges and future directions. Quantitative Plant Biology, 3, E5. doi: [10.1017/qpb.2021.18](https://doi.org/10.1017/qpb.2021.18).
- Pucker B (2022). Automatic identification and annotation of MYB gene family members in plants. BMC Genomics 23, 220 (2022). doi: [10.1186/s12864-022-08452-5](https://doi.org/10.1186/s12864-022-08452-5).
- Rempel A., Choudhary N. and Pucker B. (2023). KIPEs3: Automatic annotation of biosynthesis pathways. PLOS ONE 18(11): e0294342. doi: [10.1371/journal.pone.0294342](https://doi.org/10.1371/journal.pone.0294342).
